src/renderer/components/Commit.vue

```diff
@@ -47,7 +47,7 @@
           <h2 class="underlined">{{$t('myWeight')}}</h2>
           <div class="highlight">{{$formatNumber($store.state.commitState.weightPercentage)}}%</div>
           <div class="lower"><span>{{$t('myWeight')}}</span>{{$formatNumber($store.state.commitState.userWeight)}}</div>
-          <div class="lower"><span>{{$t('networkWeight')}}</span>{{$formatNumber($store.state.commitState.networkWeight)}}</div>
+          <div class="lower"><span>{{$t('networkWeight')}}</span>{{$abbreviateNumber($store.state.commitState.networkWeight)}}</div>
```

Comment:
I know that this file had this before you touched it, but can you fix all text interpolation with proper spacing? We are trying to fix this up as we come across it. Thanks!

```
// Old way
{{$abbreviateNumber($store.state.commitState.networkWeight)}}

// New way
{{ $abbreviateNumber($store.state.commitState.networkWeight) }}
```